src/main/java/calculator/controller/CalculateController.java

```diff
@@ -0,0 +1,30 @@
+package calculator.controller;
+
+import calculator.model.NumberFinder;
+import calculator.model.Mathematician;
+import calculator.model.Calculator;
+import calculator.model.DelimiterManager;
+import calculator.view.InputView;
+import calculator.view.OutputView;
+
+import static calculator.view.enumerate.Phrase.RESULT_PRINT_SENTENCE;
+import static calculator.view.enumerate.Phrase.START_PRINT_SENTENCE;
+
+public class CalculateController {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final Calculator calculator;
+
+    public CalculateController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.calculator = new Calculator(new Mathematician(), new NumberFinder(new DelimiterManager()));
+    }
+
+    public void run() {
+        outputView.print(START_PRINT_SENTENCE);
+        String input = inputView.startInputValue();
+        int result = calculator.sumOfNumInAlphaNum(input);
+        outputView.print(RESULT_PRINT_SENTENCE,result);
```

Comment:
```suggestion
        outputView.print(RESULT_PRINT_SENTENCE, result);
```

쉼표 뒤에는 띄워주시면 더 좋을 것 같아요! (이미 하셨을 수도 있지만,) 프리코스 커뮤니티에서 쉽게 찾을 수 있는 우테코 코드 스타일을 적용하신다면 저장과 함께 자동으로 포맷팅 하실 수 있어요!